src/lib/keychain/insecure.js

```diff
@@ -12,15 +12,15 @@ import path from 'path';
  */
 import type { Keychain } from './keychain';
 
-export default class Secure implements Keychain {
+export default class Insecure implements Keychain {
 	file: string;
 
 	constructor( file: string ) {
 		// only current user has read-write access
 		const rw = 0o600;
 
 		let stat;
-		const tmpfile = os.tmpdir() + path.sep + file;
+		const tmpfile = os.homedir() + path.sep + file;
```

Comment:
Should we store it in a subdirectory like `.vip`?

Reply:
Sounds like a good idea.